.github/workflows/github-cxx-qt-tests.yml

```diff
@@ -7,6 +7,8 @@
 name: cxx-qt tests
 on:
   push:
+    branches:
+      - 'main'
```

Comment:
do we also want it to run on stable branches? So `0.5.x` and `0.6.x` etc? 

We already run the book on a tag with `    tags: v[0-9]+\.[0-9]+\.[0-9]+` wonder if we can have like
```
branches:
   - 'main'
   - [0-9]+\.[0-9]+\.\x
```

Reply:
Done

Reply:
Assume it works :-)